lib/src/widgets/utils/ds_header.widget.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+class DSHeader extends StatelessWidget implements PreferredSizeWidget {
+  final String title;
+  final String? subtitle;
+  final String? customerName;
+  final List<Widget>? actions;
+  final Widget? leading;
+
+  const DSHeader({
+    Key? key,
+    required this.title,
+    this.subtitle,
+    this.customerName,
+    this.actions,
+    this.leading,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return AppBar(
+      elevation: 1,
+      backgroundColor: DSColors.neutralLightSnow,
```

Comment:
Looking at the prototype, the bottom shadow color is set to ```Product/$color-neutral-medium-wave```, so I think you should do the same.

```suggestion
      backgroundColor: DSColors.neutralLightSnow,
      shadowColor: DSColors.neutralMediumWave,
```

Reply:
done